akka-docs/rst/java/http/routing-dsl/directives/basic-directives/extractDataBytes.rst

```diff
@@ -0,0 +1,16 @@
+.. _-extractDataBytes-java-:
```

Comment:
those need to be added to the list of directives (see index.rst) otherwise they won't show up